src/com/robibp/trening/UnosKlinca.java

```diff
@@ -0,0 +1,37 @@
+package com.robibp.trening;
+
+import java.util.Scanner;
+
+class UnosKlinca {
```

Comment:
Ova klasa je u puno boljem stanju nego prosli puta ali jos uvijek imas bug. Imam osjecaj kao da nisi isprobao upisati npr 3 puta ime sa brojevima pa onda tocno, da vidis jel zapravo radi to sta zelis.

Uglavnom imas bug, ovaj puta malo drugaciji. Kada ga popravis cu ja tebi napraviti Pull Request u kojem cu malo pocistiti i presloziti stvari, nista posebno pa ces ti to pregledat. Zasada samo testiraj prije nego pushas i rijesi taj bug